org.mixedrealitytoolkit.uxcomponents/CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [3.2.0-development] - 2024-02-04
```

Comment:
No need to update the minor version.  The minor version has already been updated.

Before updating versions, check what version was last released: 

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases

Reply:
Note, in the future, if you do need to add information to the changelog for a new version, add a NEW section. Do not edit existing section titles.  The changelog must contain changes for ALL versions.

Note, the changelog files are new...hence why they are empty :)

Reply:
Updated in latest commit.